doc/src/guides/solving/solve-system-of-equations-algebraically.md

```diff
@@ -48,9 +48,9 @@ tuple):
 
 ### Use a Dictionary of Solutions
```

Comment:
Terminology clarification here: a single dictionary is a solution (singular) for all of the variables. Each dictionary in the list of solution-dictionaries is a solution. So maybe "Using a solution given as a dictionary" or something similar...